build/build_app_engine_package.py

```diff
@@ -15,6 +17,39 @@
 DEST_PATH    Path to the root directory to push/deploy GAE from."""
 
 
+def call_cmd_and_return_output_lines(cmd):
+  process = subprocess.Popen(cmd, stdout=subprocess.PIPE)
+  output = process.communicate()[0]
+  return output.split('\n')
+
+
+def build_version_info_file(dest_path):
+  """Build the version info JSON file."""
+  version_info = {
+      'gitHash': None,
```

Comment:
Think it's better to not initialize and let block below add entries. Don't know what happens when json.dumps gets a None value.